mne/beamformer/_rap_music.py

```diff
@@ -168,7 +168,7 @@ def _compute_subcorr(G, phi_sig):
     # in G and handle the fact that it might be rank defficient
     # eg. when using MEG and a sphere model for which the
     # radial component will be truly 0.
-    rank = np.sum(Sg > (Sg[0] * 1e-12))
+    rank = np.sum(Sg > (Sg[0] * 1e-6))
```

Comment:
At this point in the code, is `Sg` whitened or rescaled by channel type? If not, this could cause problems for M-EEG data, as the gain matrix could differ by a factor this large IIRC.

Reply:
Yes at this point, G (so Sg) is already whitened